app/components/e-heading.vue

```diff
@@ -0,0 +1,164 @@
+<!-- This component has no <template> because of dynamic root element -->
+
+<script>
+  export default {
+    name: 'e-heading',
+    // components: {},
+    // mixins: [],
+
+    props: {
+      /**
+       * Defines the color of the heading (h1-h6)
+       */
+      color: {
+        type: String,
+        default: 'blue',
+        validator(value) {
+          return [
+            'blue',
+            'green',
+            'darkgrey',
+          ].includes(value);
+        },
+      },
+
+      /**
+       * Defines the level of the heading (h1-h6)
+       */
+      level: {
```

Comment:
If it is called "level", please use Number as type and just pass the numbers 1-6

Reply:
Re-named it to tagName :)